Bugzilla/API/V1/Configuration.pm

```diff
@@ -0,0 +1,273 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+
+package Bugzilla::API::V1::Configuration;
+
+use 5.10.1;
+
+use Bugzilla::Constants;
+use Bugzilla::Field;
+use Bugzilla::Keyword;
+use Bugzilla::Logging;
+use Bugzilla::Product;
+use Bugzilla::Status;
+
+use List::MoreUtils qw(any);
+use Mojo::Base qw( Mojolicious::Controller );
+use Mojo::JSON qw( true false );
+
+sub setup_routes {
+  my ($class, $r) = @_;
+  $r->get('/latest/configuration')->to('V1::Configuration#configuration');
```

Comment:
It's weird that the versioning is _internal_ only; the main point of versioning an API is for _external_ consumers – eg `/rest/v1/bug/12345` vs `/rest/v2/bug/12345`; that way consumers are isolated from breaking changes.

As it stands I'm unclear on the benefit on this versioning work.

Reply:
/rest/bug is the same as /rest/v1/bug as the original API paths are always considered v1. For v2 and beyond, people would then need to start including versions in the path.